lib/webmock/http_lib_adapters/httpclient_adapter.rb

```diff
@@ -105,6 +103,44 @@ def build_httpclient_response(webmock_response, stream = false, req_header = nil
     end
   end
 
+  class WebMockHTTPClient < HTTPClient
+    alias_method :do_get_block_without_webmock, :do_get_block
+    alias_method :do_get_stream_without_webmock, :do_get_stream
+
+    include WebMockHTTPClients
+  end
+
+  if defined? ::JSONClient
```

Comment:
It seems there is duplication here. HTTPClientAdapter with enable and disable methods has already been defined and now it's being redefined.

Is there a reason to define `HTTPClientAdapter` twice in case JSONClient is defined? Perhaps the JSONClient related code can be wrapped in `if defined? ::JSONClient`?